CRM/Price/Form/Option.php

```diff
@@ -216,6 +216,8 @@ public function buildQuickForm() {
       $this->addRule('amount', ts('Please enter a monetary value for this field.'), 'money');
 
       $this->add('textarea', 'description', ts('Description'));
+      $this->add('textarea', 'help_pre', ts('Pre Option Help'), NULL, TRUE);
```

Comment:
I'm guessing the problem is in the postProcess on this file? You shouldn't need to do anything re multilingual - esp if you use the api